timestamp.js

```diff
@@ -0,0 +1,34 @@
+const Http = require("http");
+
+const server = Http.createServer((req, res) => {
+
+    const timeStampStartIndex = req.url.indexOf(":");
```

Comment:
did not get this logic, req should have data object in it which would contain parameters passed.

Reply:
<img width="756" alt="Screen Shot 2019-03-31 at 9 26 44 AM" src="https://user-images.githubusercontent.com/46927470/55284559-248e8d80-5397-11e9-8b74-877835f63682.png">

They asked to send the parameter like this